addons/europeans_tribe.wad/addon

```diff
@@ -2,7 +2,7 @@
 name=_"Europeans Tribe"
 description=_"Adds a new tribe named ‘Europeans’. It’s still experimental and uses buildings and workers from Barbarians, Atlanteans, and Empire, until own graphics are available."
 author="MarkMcWire"
-version="1.6.1"
+version="1.6.1f"
```

Comment:
```suggestion
version="1.6.2"
```
Version strings should consist only of numbers separated by fullstops. Everything else is ignored.